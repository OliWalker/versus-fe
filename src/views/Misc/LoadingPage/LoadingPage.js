import React from 'react';
import './LoadingPage.css';

const Loading = () => {
  return (
    <div className="LoadingPage">
      <div className="lds-ripple">
        <div />
        <div />
      </div>
    </div>
  );
};

export default Loading;
