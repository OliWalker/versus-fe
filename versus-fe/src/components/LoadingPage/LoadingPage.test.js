import React from 'react';
import { shallow } from 'enzyme';
import LoadingPage from './LoadingPage';
import toJson from 'enzyme-to-json';

test('LoadingPage should render as expected', () => {
  const component = shallow(<LoadingPage />);
  const tree = toJson(component);
  expect(tree).toMatchSnapshot();
});
