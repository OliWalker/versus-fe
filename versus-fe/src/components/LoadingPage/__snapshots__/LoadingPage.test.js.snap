// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Page LoadingPage should render as expected 1`] = `
<div
  className="LoadingPage"
>
  <div
    className="lds-ripple"
  >
    <div />
    <div />
  </div>
</div>
`;

exports[`LoadingPage should render as expected 1`] = `
<div
  className="LoadingPage"
>
  <div
    className="lds-ripple"
  >
    <div />
    <div />
  </div>
</div>
`;
